source/dem/particle_particle_contact_force.cc

```diff
@@ -1,1284 +1,230 @@
 #include <core/parameters_lagrangian.h>
-#include <core/tensors_and_points_dimension_manipulation.h>
 
 #include <dem/particle_particle_contact_force.h>
 
 using namespace DEM;
+using namespace Parameters::Lagrangian;
 
-template <
-  int                                                       dim,
-  Parameters::Lagrangian::ParticleParticleContactForceModel contact_model,
-  Parameters::Lagrangian::RollingResistanceMethod rolling_friction_model>
+template <int                               dim,
+          ParticleParticleContactForceModel contact_model,
+          RollingResistanceMethod           rolling_friction_model>
 ParticleParticleContactForce<dim, contact_model, rolling_friction_model>::
   ParticleParticleContactForce(const DEMSolverParameters<dim> &dem_parameters)
   : dmt_cut_off_threshold(dem_parameters.model_parameters.dmt_cut_off_threshold)
 {
   set_effective_properties(dem_parameters);
 }
 
-template <
-  int                                                       dim,
-  Parameters::Lagrangian::ParticleParticleContactForceModel contact_model,
-  Parameters::Lagrangian::RollingResistanceMethod rolling_friction_model>
+template <int                               dim,
+          ParticleParticleContactForceModel contact_model,
+          RollingResistanceMethod           rolling_friction_model>
 void
 ParticleParticleContactForce<dim, contact_model, rolling_friction_model>::
   calculate_particle_particle_contact_force(
-    DEMContactManager<dim>    &container_manager,
+    typename dem_data_structures<dim>::adjacent_particle_pairs
+      &local_adjacent_particles,
+    typename dem_data_structures<dim>::adjacent_particle_pairs
+      &ghost_adjacent_particles,
+    typename dem_data_structures<dim>::adjacent_particle_pairs
+      &local_periodic_adjacent_particles,
+    typename dem_data_structures<dim>::adjacent_particle_pairs
+      &ghost_periodic_adjacent_particles,
+    typename dem_data_structures<dim>::adjacent_particle_pairs
+                              &ghost_local_periodic_adjacent_particles,
```

Comment:
Do that (as it used to be before the implementation of the contact manager) so it is more clear what is neededin this function (also will be the same as for the pw contact force)